### DISCUSSION TITLE
Edge detection filters

### DISCUSSION THREAD

**Discussion by rafaelblevin821** [BindsNET/bindsnet#587] [General]
Hi, just a quick question because I can't find anything in the documentation.

Does BindsNet have any built-in edge detection filters such as Sobel or Canny? I have a dataset of ultrasound images which are very obscure and could benefit from a filter before going through the network.

If not, it's not big deal, I can implement my own or import one from pytorch.

**Comment by Hananel-Hazan** [comment]
BindsNET dont have this filter build-in, though it could be a nice addition. If you end up implement one, let me know and we can think on a way to add it to BindsNET with your contribution.

**Reply by rafaelblevin821** [reply]
Hi @Hananel-Hazan , no worries, thanks for letting me know. I will get back to you with the progress.
